components/VM_Arm/plat_include/tx2/plat/vmlinux.h

```diff
@@ -164,7 +164,7 @@ static const int free_plat_interrupts[] = { 220 + GIC_LIC_INTID_BASE };
 #define DTB_ADDR          (LINUX_RAM_BASE + 0x01000000)
 #define INITRD_MAX_SIZE   0x1900000 //25 MB
 #define INITRD_ADDR       (DTB_ADDR - INITRD_MAX_SIZE) //0x80700000
-#define GIC_IRQ_PHANDLE 0x1
```

Comment:
This means that the GIC_NODE_PATH has to match with the name in the DTS, right?

Reply:
Correct.